docs/book/v6/tutorials/create-book-module.md

```diff
@@ -642,78 +735,46 @@ declare(strict_types=1);
 
 namespace Api\Book;
 
-use Api\Book\Collection\BookCollection;
-use Api\Book\Entity\Book;
-use Api\Book\Handler\BookHandler;
-use Api\Book\Repository\BookRepository;
-use Api\Book\Service\BookService;
-use Api\Book\Service\BookServiceInterface;
-use Doctrine\ORM\Mapping\Driver\AttributeDriver;
-use Dot\DependencyInjection\Factory\AttributedRepositoryFactory;
-use Dot\DependencyInjection\Factory\AttributedServiceFactory;
+use Api\Book\Handler\GetBookCollectionHandler;
+use Api\Book\Handler\GetBookHandler;
+use Api\Book\Handler\PostBookHandler;
+use Core\App\ConfigProvider;
+use Dot\Router\RouteCollectorInterface;
 use Mezzio\Application;
-use Mezzio\Hal\Metadata\MetadataMap;
-use Api\App\ConfigProvider as AppConfigProvider;
+use Psr\Container\ContainerExceptionInterface;
+use Psr\Container\ContainerInterface;
+use Psr\Container\NotFoundExceptionInterface;
 
-class ConfigProvider
+class RoutesDelegator
 {
-    public function __invoke(): array
+    /**
+     * @throws ContainerExceptionInterface
+     * @throws NotFoundExceptionInterface
+     */
+    public function __invoke(ContainerInterface $container, string $serviceName, callable $callback): Application
```

Comment:
As long as it's not justified to use groups, we should keep things simple because this way it's easier for devs to follow the tutorial.

Replace:

```php
$routeCollector->group('/book')->post('', PostBookHandler::class, 'book::create-book');
$routeCollector->group('/book/' . $uuid)->get('', GetBookHandler::class, 'book::view-book');
$routeCollector->group('/books')->get('', GetBookCollectionHandler::class, 'book::list-books');
```

with:

```php
$routeCollector->post('/book', PostBookHandler::class, 'book::create-book');
$routeCollector->get('/book/' . $uuid, GetBookHandler::class, 'book::view-book');
$routeCollector->get('/books', GetBookCollectionHandler::class, 'book::list-books');
```